app/uk/gov/hmrc/helptosavefrontend/views/reminder/reminder_cancel_confirmation.scala.html

```diff
@@ -34,7 +34,7 @@
 
     <h2 class="govuk-heading-m">@Messages("hts.reminder-confirmation.h2")</h2>
 
-    <p class="govuk-body">@Messages("hts.reminder-cancel-confirmation.p1-1") <a class="govuk-link" href=@routes.ReminderController.getEmailsavingsReminders.url>
+    <p class="govuk-body">@Html(Messages("hts.reminder-cancel-confirmation.p1-1")) <a class="govuk-link" href=@routes.ReminderController.getEmailsavingsReminders.url>
```

Comment:
we don't need to wrap this message in Html - when I look at the message value it is a plain string (with no html tags within) that says "You can" in English and "Gallwch" in Welsh - worth checking sometimes if we see it in the old code we are upgrading that Html parsing is required - unnecessary or blanket Html wrapping is a [security concern ](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=TEC&postingDay=2023%2F1%2F23&title=XSS+Protection+in+play+frontends#XSSProtectioninplayfrontends-play.twirl.api.Html)so we are encouraged to use only where necessary and we can confirm the html is safe